### PR TITLE
Feature/forms branch - Add forms to new and edit routes 

### PR DESCRIPTION
**Pull request by ricardovaltierra**
The following features where added:

- Back End set up
  * Create User model w/ username, email and password
  * Migrate db
  * Validations for presence
  * Views created for 'new' and 'edit' actions with `form_for`

### CODE REVIEW

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required :red_circle: 

Hello @ricardovaltierra @ClintonEnyinna 

Everything is almost OK except the forms.
According to [that](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form1.png), [that one too](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form2.png), and [this last one](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form_tag2.png), you need to implement the form. 
And after doing it, you have to comment them except the last one. We will check the commented code.
You have to do it form `new` and `edit` path.

Kindly make the suggested changes and submit a new code review request
Thanks

Mael FOSSO

File: app/views/users/edit.html.erb [hunk 1]
```diff
@@ -6,12 +6,43 @@
...
+<%= form_tag("/users", method: "post") do %>
+  <%= label_tag(:username, "Username:") %>
+  <%= text_field_tag "user[username]", @user.username %>
+  <br>
+  <%= label_tag(:email, "Email:") %>
+  <%= text_field_tag "user[email]", @user.email %>
+  <br>
+  <%= label_tag(:password, "Password:") %>
+  <%= password_field_tag "user[password]", @user.password %>
+  <br>
```
Comment (samgaco) [inline app/views/users/edit.html.erb:47]: this should be commented too

File: app/views/users/new.html.erb [hunk 2]
```diff
@@ -1,9 +1,41 @@
...
+-->
+
+<!--
+<%= form_tag("/users", method: "post") do %>
+  <%= label_tag(:username, "Username:") %>
+  <%= text_field_tag "user[username]" %>
+  <br>
+  <%= label_tag(:email, "Email:") %>
+  <%= text_field_tag "user[email]" %>
+  <br>
```
Comment (samgaco) [inline app/views/users/new.html.erb:35]: should be commented

**Review by samgaco — APPROVED** [review]
Hi @ricardovaltierra and @ClintonEnyinna :high_brightness: 

I left some comments below some fixes you should handle before merging.

The project is approved :+1: 

-------------------------

* ###### STATUS OF THE PROJECT: APPROVED :heavy_check_mark:

**Pull request merged**